Dockerfile

```diff
@@ -15,18 +15,19 @@ LABEL "maintainer"="Matt Brown <github@muglug.com>"
 
 RUN apk add --no-cache tini git
 
-COPY --from=composer:1.9 /usr/bin/composer /usr/bin/composer
+COPY --from=composer:2 /usr/bin/composer /usr/bin/composer2
+COPY --from=composer:1 /usr/bin/composer /usr/bin/composer1
```

Comment:
Is it still needed in 2021 to expose the old composer version? I'd rather expose only the versions 2.0 and simplify/lighten this Image.

Reply:
I wasn't sure either, but I thought that maybe people would like to keep composer1 for some stuff (e.g not having a composer plugin compatible with composer 2).

Unless we could ask them to use an old version of this action ?

Reply:
My 2 cents: People can use an older version of this Image, or use a previous step in github workflow that run composer install from the composer version they want.

Let's spread the adoption of composer v2.

Reply:
Removed that part then, forcing to use composer2 at all time.